docset/winserver2022-ps/failoverclusters/Remove-ClusterCheckpoint.md

```diff
@@ -36,20 +36,13 @@ local server.
 
 ### Example 1
 
+```powershell
+Get-ClusterResource "Cluster Name" | Remove-ClusterCheckpoint -RegistryCheckpoint
+Remove-ClusterCheckpoint -Confirm:$false
 ```
-PS C:\> Get-ClusterResource "Cluster Name" | Remove-ClusterCheckpoint -RegistryCheckpoint
 
-
-
-PS C:\> Remove-ClusterCheckpoint
-Are you sure you want to remove registry checkpoint 'software\clusname' on resource 'Cluster Name'? 
-
-
-[Y] Yes  [N] No  [S] Suspend  [?] Help (default is "Y"):Y
-```
-
-This example removes the registry checkpoint called software\clusname for the resource named Cluster
-Name.
+This example removes the registry checkpoint called `software\clusname` for the resource named
+`Cluster Name` without user confirmation.
```

Comment:
Without the output, it's not clear to me where `software\clusname` is coming from, which might be confusing for a reader.

Reply:
Good point, plus the command didn't make sense. Updated.